src/libs/DateUtils.ts

```diff
@@ -360,12 +360,19 @@ function getMicroseconds(): number {
     return Date.now() * CONST.MICROSECONDS_PER_MS;
 }
 
+/**
+ * Returns the format yyyy-MM-dd HH:mm:ss of a date in the format expected by the database
```

Comment:
Okey, let's nuke this comment (as it's a helper function related to `getDBTime`) and update the original comment so it makes sense.

Reply:
👉 https://github.com/Expensify/App/pull/34784#discussion_r1466183362